packages/eslint-plugin-turbo/lib/rules/no-undeclared-env-vars.ts

```diff
@@ -177,6 +175,13 @@ function create(context: RuleContextWithOptions): Rule.RuleListener {
           ) {
             checkKey(node.parent, node.parent.property.name);
           }
+        } else if (
+          "property" in node.parent &&
+          node.parent.property.type === "Literal" &&
+          typeof node.parent.property.value === "string"
```

Comment:
Does this capture template strings too? I see the `type` in astexplorer is `TemplateLiteral`, but I'm not sure if the eslint parser is the same
 https://astexplorer.net/#/gist/b8b068eb786edac5abd35572cdf791e7/de84c6402467ab65b0b6d71a15f5f320723f655b, so

Reply:
I don't believe so. We could extend it to capture templates that are only literals if we really care to, but at some point we're running into calculated values that we can't really inspect.